hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/utils/TestSQL.java

```diff
@@ -42,6 +42,13 @@ private TestSQL() {
       + "('id1','Danny',23,TIMESTAMP '1970-01-01 00:00:02','par1'),\n"
       + "('id1','Danny',23,TIMESTAMP '1970-01-01 00:00:01','par1')";
 
+  public static final String INSERT_SAME_KEY_T2 = "insert into t1 values\n"
+          + "('id1','Danny',23,TIMESTAMP '1970-01-01 00:00:04','par1'),\n"
+          + "('id1','Danny',23,TIMESTAMP '1970-01-01 00:00:03','par1'),\n"
```

Comment:
Can this SQL be avoided if we include the fifth item for `DATA_SET_DISORDER_INSERT` ?

Reply:
Sorry, this is not used,I forgot to delete it.